### PR TITLE
Extensions of inspector window (#5867)

### PR DESCRIPTION
**Pull request by rfranke**
This reduces CPU load if the inspector window is not used.

### CODE REVIEW

File: rtgui/inspector.cc [hunk 1]
```diff
@@ -166,6 +171,25 @@ bool Inspector::on_button_press_event(GdkEventButton *event)
     return false;
 }
 
+bool Inspector::on_motion_notify_event(GdkEventMotion *event)
+{
```
Comment (Lawrence37) [inline rtgui/inspector.cc:218]: Make sure `currImage` is not null here (and also in `Inspector::on_zoom_begin`, I think).

Comment (rfranke) [reply rtgui/inspector.cc:218]: Thank you for the code review. Added two checks to 947f3dc.

**Comment by Thanatomanic** [comment]
@rfranke What is the status of this? In particular the commit that fixes the CPU usage would be very good to merge.

**Comment by rfranke** [comment]
Some of the other commits might be important as well, like panning with mouse drag and F11 to toggle the fullscreen status. Moreover, the extension of the filebrowser context menu makes the inspector window accessible through the GUI (cat mode ;) ).

Shall I revert the last commit as you don't like the either .. or decision?

**Comment by heckflosse** [comment]
@rfranke btw: please excuse that I introduced a merge conflict with my latest commit :frowning_face:

**Comment by ff2000** [comment]
> Shall I revert the last commit as you don't like the either .. or decision?

I would vote for "keep it". Having the context menu entry (that IMO is important to discover the feature) and a visible "Inspector" tab nearby might totally confuse people when a new window opens instead of the image being shown in the tab. IMO it really is an either-or thing.

**Comment by heckflosse** [comment]
What's the status of this PR?
In my opinion we have to merge this for 5.9 because 5.9 filebrowser will become unusable when working with raw files from smartphones, which do not have an embedded jpg, if we do not merge.

**Pull request merged**